components/developer/jenkins/files/jenkins-standalone.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version='1.0'?>
+<!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
+<service_bundle type='manifest' name='jenkins-standalone'>
+  <service
+    name='network/http'
```

Comment:
Well, it does present an HTTP server when run standalone, and end-users might want to be aware of that. In particular, some orgs did have to close down public access to Jenkins farms back in the day, and just republish build results and logs as static data etc. when they suspected or found vulnerabilities in Jenkins app itself. Also, using a relatively common port of 8080, this one can conflict on systems that do not run services in separate zones...